omero_search_engine/api/v1/resources/utils.py

```diff
@@ -291,8 +312,20 @@ def elasticsearch_query_builder(
             search_omero_app.logger.info("FILTER %s" % filter)
             try:
                 key = filter["name"].strip()
-                value = filter["value"].strip()
                 operator = filter["operator"].strip()
+                if operator == "in" or operator == "not_in":
```

Comment:
if we had other operators laters , a list will be easier to handle 
i.e. if operator in l then do something